feature-libs/product/multi-dimensional/public_api.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './index';
```

Comment:
[MAJOR]
As we agreed on the call, let's move Multi-D feature to a standalone feature lib - because this lib will contribute to multiple features/views (PDP, PLP, Cart). What about name `@sparatcus/product-multidimentional` ?

Note: it might affect, among other things, also the schematics

Reply:
From what I understand, you don't want it to be a sub-library of the product library?

Reply:
Correct. I meant to not create a sub-library of product lub, but instead: a standalone library.
Any objections? I'm happy to learn your point of view.

Reply:
I've moved it to the standalone library in a separate pr https://github.com/SAP/spartacus/pull/19083/files